showpass/Showpass-E2E-tests
# TA-26-Profile -Verify the 'Change Password' inline validation

**Description:**
This spec will be verifying that proper inline validation messages show up when an end user is providing invalid input while changing password.
Steps:
- Navigate to https://beta.showpass.com/account/profile/change-password/
- Do not populate the fields
- Click ‘Save’ right away
- Verify that proper error messages show up under each input field

**Jira ticket:**
https://showpass.atlassian.net/browse/TA-26

**How to test:**
Run the following test suite and ensure the corresponding test passes
npx cypress run --spec "cypress/e2e/user-settings/profile.cy.js"

**Test execution results:**

![TA-26-passed](https://github.com/showpass/Showpass-E2E-tests/assets/138615449/9e86781b-37fc-4eed-bc94-a62132b2867a)
